modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -2375,6 +2375,52 @@ m|+++false+++
 |===
 
 
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_apply_parallelism]]
+=== `db.index.fulltext.eventually_consistent_apply_parallelism`
+
+.db.index.fulltext.eventually_consistent_apply_parallelism
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|The number of threads that are applying the queued index updates for eventually consistent full-text indexes.
+|Valid values
+a|An integer that is minimum 1.
+|Default value
+m|+++1+++
+|===
+
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_refresh_interval]]
+=== `db.index.fulltext.eventually_consistent_refresh_interval`
+
+.db.index.fulltext.eventually_consistent_refresh_interval
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|How often an eventually consistent full-text index will be refreshed (readers guaranteed to see changes). If set to `0` then refresh is done by the threads applying eventually consistent fulltext index updates.
```

Comment:
```suggestion
a|How often an eventually consistent full-text index is refreshed (readers are guaranteed to see changes). If set to `0`, refresh is done by the threads applying eventually consistent full-text index updates.
```

(readers are guaranteed to see changes) sounds strange to me for a config description. Who are the readers?

Reply:
Same here - needs to be merged in the codebase before the cut-off.